src/lib-voyager.tsx

```diff
@@ -0,0 +1,86 @@
+// This module is intended to be used when embedding voyager
+// in some other context than the orgiginal app.
+//
+// It provides factory methods for creating instances of the Voyager application
+// and should eventually also export a react component if one is doing that kind
+// of integration.
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+import 'font-awesome-sass-loader'; // TODO should this move to App?
+
+import {App, VoyagerConfig, VoyagerData} from './components/app';
+import {configureStore} from './store';
+
+type Container = string | HTMLElement;
+
+
+/**
+ * The Voyager class encapsulates the voyager application and allows for easy
+ * instantiation and interaction from non-react projects.
+ */
+class Voyager {
+  private container: HTMLElement;
+  private config: VoyagerConfig;
+  private data: VoyagerData;
+  private store: any; // TODO how to get the type/inteface for this here?
```

Comment:
I hover over configureStore() in VSCode to see the return type and try to import things accordingly.